Week1/homework/app.js

```diff
@@ -1,11 +1,145 @@
+      const placeForList = document.body;
+      placeForList.insertBefore(emptyList, placeForList.childNodes[2]);
+    };
+    placeList();
+
+    function addTitlesToList() {
+      let x;
+      for (x in bookTitles) {
+        const newListItem = document.createElement("li");
+        const titleItem = document.createTextNode(bookTitles[x]);
+        newListItem.appendChild(titleItem);
+        emptyList.appendChild(newListItem);
+      };
+    };
+    addTitlesToList();
+  };
+  createListOfBookTitles();
+  */
+
+// **1.4** Making an object
+
+  const Books = bookTitles.reduce((acc, elem) => {
+    acc[elem] = {title:``, author:``, original_language:``};
+    return acc;
+  }, {});
+
+// Adding content
+
+  Books.The_Goldfinch.title = `The Goldfinch`;
+  Books.The_Goldfinch.author =`Donna Tartt`;
+  Books.The_Goldfinch.original_language = `English`;
+
+  Books.A_Garden_of_Earthly_Delights.title = `A Garden of Earthly Delights`;
+  Books.A_Garden_of_Earthly_Delights.author = `Carol Oates`;
+  Books.A_Garden_of_Earthly_Delights.original_language = `English`;
+
+  Books.Beloved.title = `Beloved`;
+  Books.Beloved.author =`Toni Morrison`;
+  Books.Beloved.original_language = `English`;
+
+  Books.There_Once_Lived_a_Woman_Who_Tried_to_Kill_Her_Neighbors_Baby.title = `There Once Lived a Woman Who Tried to Kill Her Neighbo\`s Baby`;
+  Books.There_Once_Lived_a_Woman_Who_Tried_to_Kill_Her_Neighbors_Baby.author = `Lyudmila Petrushevskaya`;
+  Books.There_Once_Lived_a_Woman_Who_Tried_to_Kill_Her_Neighbors_Baby.original_language = `Russian`;
+
+  Books.The_House_of_the_Spirits.title = `The House of the Spirits`;
+  Books.The_House_of_the_Spirits.author = `Isabel Allende`;
+  Books.The_House_of_the_Spirits.original_language = `Spanish`;
+
+  Books.The_Left_Hand_of_Darkness.title = `The Left Hand of Darkness`;
+  Books.The_Left_Hand_of_Darkness.author = `Ursula K. Le Guin`;
+  Books.The_Left_Hand_of_Darkness.original_language = `English`;
+
+  Books.The_Unwomanly_Face_of_War.title = `The Unwomanly Face of War`;
+  Books.The_Unwomanly_Face_of_War.author = `Svetlana Alexievich`;
+  Books.The_Unwomanly_Face_of_War.original_language = `Russian`;
+
+  Books.Chernobyl_Prayer.title = `Chernobyl Prayer`;
+  Books.Chernobyl_Prayer.author = `Svetlana Alexievich`;
+  Books.Chernobyl_Prayer.original_language = `Russian`;
+
+  Books.Runaway.title = `Runaway`;
+  Books.Runaway.author = `Alice Munro`;
+  Books.Runaway.original_language = `English`;
+
+  Books.The_Year_of_Magical_Thinking.title = `The Year of Magical Thinking`;
+  Books.The_Year_of_Magical_Thinking.author = `Joan Didion`;
+  Books.The_Year_of_Magical_Thinking.original_language = `English`;
+
+  console.log(Books);
+  
+  // **1.5** function that take the actual information about the book from the object
+  
+  const booksList = document.createElement(`ul`);
+  document.body.insertBefore(booksList, document.body.childNodes[2]);
+  
+    function placeContent() {
+      let x;
+      for (x in Books) {
```

Comment:
loop over the bookTitles and access each book from the Books object:

Example:
```
const fruitArray = ["banana", "orange", "apple", "lime"];

const fruitObject = {
  banana: {
    color: "yellow",
    weight: 200,
  },
  orange: {
    color: "orange",
    weight: 150,
  },
  apple: {
    color: "red",
    weight: 120
  },
  lime: {
    color: "green",
    weight: 60
  }
};

for (const fruit of fruitArray) {
  const color = fruitObject[fruit].color;
  const weight = fruitObject[fruit].weight;

  console.log(color);
  console.log(weight);
}
```

Reply:
done